demosplan/DemosPlanCoreBundle/Addon/AddonRoutingLoader.php

```diff
@@ -36,7 +36,9 @@ public function loadRoutes(): RouteCollection
     {
         $routeCollection = new RouteCollection();
         foreach ($this->addonRegistry->getAddonInfos() as $addonInfo) {
-            $this->addControllers($addonInfo, $routeCollection);
+            if ($addonInfo->isEnabled()) {
```

Comment:
For me, it was enough to clear the cache after enabling the addon to force a frontend-build, which we have to do anyway after enabling an addon.

Also, we need to check if an addon is enabled first as we would otherwise add functional routes that are not allowed but available.

Reply:
I tested it myself again - this works fine.